db/migrate/20160825213523_change_load_balancer_listener_port_to_port_range.rb

```diff
@@ -0,0 +1,55 @@
+class ChangeLoadBalancerListenerPortToPortRange < ActiveRecord::Migration[5.0]
+  def change
+    reversible do |dir|
```

Comment:
@selmanj is there reason for using `reversible`? can we use methods `def up` and `def down` ? as we have in other migrations ?